### PR TITLE
chore: Fix minor TODOs and add tests

### PR DESCRIPTION
**Pull request by dgusakov**

### CODE REVIEW

File: src/CSModule.sol [hunk 1]
```diff
@@ -74,7 +75,8 @@ contract CSModule is
     mapping(uint256 noKeyIndexPacked => bool) private _isValidatorWithdrawn;
     mapping(uint256 noKeyIndexPacked => bool) private _isValidatorSlashed;
 
-    TransientUintUintMap private _queueLookup; // TODO: Add explainer comment
+    // @dev transient map used in by cleanDepositQueue method to keep track of already oserved queue batches contents
```
Comment (skhomuti) [inline src/CSModule.sol:78]:
```suggestion
    // @dev transient map used in by cleanDepositQueue method to keep track of already observed queue batches contents
```

File: src/CSModule.sol [hunk 2]
```diff
@@ -869,7 +866,7 @@ contract CSModule is
         onlyRole(STAKING_ROUTER_ROLE)
     {
         // solhint-disable-previous-line no-empty-blocks
-        // Nothing to do, rewards are distributed by a performance oracle.
+        // Nothing to do, rewards are distributed by a performance ora`.
```
Comment (skhomuti) [inline src/CSModule.sol:869]:
```suggestion
        // Nothing to do, rewards are distributed by a performance oracle.
```

**Review by skhomuti — CHANGES_REQUESTED** [review]

**Review by skhomuti — APPROVED** [review]

File: test/CSAccounting.t.sol [hunk 3]
```diff
@@ -35,6 +36,80 @@ import { ERC20Testable } from "./helpers/ERCTestable.sol";
...
+            address(locator),
+            address(stakingModule),
+            10,
+            4 weeks,
+            365 days
+        );
+        assertEq(address(accounting.CSM()), address(stakingModule));
+    }
+
+    function test_constructor_canNotInit() public {
```
Comment (vgorkavenko) [inline test/CSAccounting.t.sol:79]:
```suggestion
    function test_constructor_RevertWhen_InitOnImpl() public {
```

or smth like that

File: test/CSEarlyAdoption.t.sol [hunk 4]
```diff
@@ -24,17 +25,53 @@ contract CSEarlyAdoptionTest is Test, Utilities {
...
-        assertEq(earlyAdoption.TREE_ROOT(), merkleTree.root());
+    function test_constructor() public {
+        earlyAdoption = new CSEarlyAdoption(root, curveId, csm);
+        assertEq(earlyAdoption.TREE_ROOT(), root);
         assertEq(earlyAdoption.CURVE_ID(), curveId);
         assertEq(earlyAdoption.MODULE(), csm);
     }
 
-    function test_verifyProof() public {
+    function test_constructor_revertIf_InvalidTreeRoot() public {
```
Comment (vgorkavenko) [inline test/CSEarlyAdoption.t.sol:38]:
```suggestion
    function test_constructor_RevertWhen_InvalidTreeRoot() public {
```

File: test/CSAccounting.t.sol (hunk 3 above)
Comment (vgorkavenko) [reply test/CSAccounting.t.sol:79]: here and after

File: test/CSEarlyAdoption.t.sol (hunk 4 above)
Comment (vgorkavenko) [reply test/CSEarlyAdoption.t.sol:38]: here and after

**Review by skhomuti — DISMISSED** [review]

**Review by skhomuti — APPROVED** [review]

**Pull request merged**